source/changenotes.ttl

```diff
@@ -0,0 +1,25 @@
+prefix skos: <http://www.w3.org/2004/02/skos/core#>
+prefix : <https://id.kb.se/vocab/>
+
+base <https://id.kb.se/>
+
+</changenote/primarycontribution> a :ChangeCategory ;
+  skos:prefLabel "Change of primary contribution"@en, "Ändring av primärt upphov"@sv .
+
+</changenote/worktitle> a :ChangeCategory ;
+  skos:prefLabel "Change of work title"@en, "Ändring av verkets titel"@sv .
+
+</changenote/primarytitle> a :ChangeCategory ;
+  skos:prefLabel "Change of primary title"@en, "Ändring av huvudtitel"@sv .
+
+</changenote/primarypublication> a :ChangeCategory ;
+  skos:prefLabel "Change of primary publication"@en, "Ändring av primär utgivning"@sv .
+
+</changenote/targetaudience> a :ChangeCategory ;
+  skos:prefLabel "Change of target audience"@en, "Ändring av målgrupp"@sv .
+
+</changenote/classification> a :ChangeCategory ;
+  skos:prefLabel "Change of classification"@en, "Ändring av klassifikation"@sv .
```

Comment:
```suggestion
</changenote/ddcclassification> a :ChangeCategory ;
  skos:prefLabel "Change of DDC classification"@en, "Ändring av DDK-klassifikation"@sv .
  
</changenote/sabclassification> a :ChangeCategory ;
  skos:prefLabel "Change of SAB classification"@en, "Ändring av SAB-klassifikation"@sv .
```
Split these for filtering purposes.
`kssbclassification` vs `sabclassification`? code is kssb, but SAB is the name in use.